packages/styles/src/mixins/_forms.scss

```diff
@@ -233,6 +233,34 @@ $fd-input-field-height--compact: 1.625rem;
   }
 }
 
+@mixin _fd-input-group-button-first-child($selector) {
+  #{$selector} {
+    @include fd-set-border-right(var(--fdInputGroup_ControlButton_SideBorder));
+
+    @include fd-hover() {
+      @include fd-set-border-right(var(--fdInputGroup_ControlButton_SideBorder_Active));
+    }
+
+    @include fd-active() {
+      @include fd-set-border-right(var(--fdInputGroup_ControlButton_SideBorder_Active));
+    }
+  }
+}
+
+@mixin _fd-input-group-button-last-child($selector) {
+  #{$selector} {
+    @include fd-set-border-left(var(--fdInputGroup_ControlButton_SideBorder));
+
+    @include fd-hover() {
+      @include fd-set-border-left(var(--fdInputGroup_ControlButton_SideBorder_Active));
+    }
+
+    @include fd-active() {
+      @include fd-set-border-left(var(--fdInputGroup_ControlButton_SideBorder_Active));
+    }
+  }
+}
+
```

Comment:
The 2 mixins can be combined into one and pass first/last as a parameter, together with the selector. Something like:
```
@mixin fd-input-group-button($selector, $child) {
    #{$selector} {
        @if($child === last) {...}
        @else {....}
    }
}

```